README.md

```diff
@@ -353,8 +353,9 @@ as follows:
 }
 ```
 
-Note, the Home Manager library is exported by the flake under
-`lib.hm`.
+Note that the `home-manager` uses its own nixpkgs input, which is updated
+periodically. See the wiki for more details.
```

Comment:
Do we have a link to the wiki section?

Reply:
Hasn't been written yet, but I don't have access to write the page.

Reply:
It should go in the manual instead of some wiki. I believe there is some work documenting flakes in HM in #1697.

Reply:
Alright, so I should just take out that sentence entirely? Or leave a reference to a to-be-written section of the manual?

Reply:
I would just remove the sentence. When we'll have docs (soon<sup>TM</sup>), we'll update the README.